content/flux/v0.x/_index.md

```diff
@@ -10,6 +10,7 @@ weight: 1
 aliases:
   - /influxdb/v2.0/reference/flux/
   - /influxdb/cloud/reference/flux/
+  - /flux/v0.65/
```

Comment:
let's discuss run through the v0.65 updates together

Reply:
@rydermeehan, per our chat, closing all PRs with Flux-related aliases. @sanderson recommends handling these redirects on the server.